### PR TITLE
LA-108 Delete document in my space presentation

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
Please review this.
Screen Shot 2021-01-20 at 17 41 09 Screen Shot 2021-01-20 at 17 41 27 Screen Shot 2021-01-20 at 17 41 39
Screen Shot 2021-01-20 at 17 49 27

### CODE REVIEW

File: lib/presentation/localizations/app_localizations.dart [hunk 1]
```diff
@@ -446,6 +446,57 @@ class AppLocalizations {
       name: 'Some items could not be copied to Shared space',
```
Comment (imGok) [inline lib/presentation/localizations/app_localizations.dart:446]: Btw just so you know the name has to be the same as the function name and you can do 
```
flutter pub run intl_translation:extract_to_arb --output-dir=lib/l10n lib/presentation/localizations/app_localizations.dart
```
so it makes the intl_messages.arb

I saw you wrote it by hand that's why! :) 

I fixed these lines in my PR tho

File: lib/presentation/localizations/app_localizations.dart [hunk 2]
```diff
@@ -446,6 +446,57 @@ class AppLocalizations {
...
     );
   }
+
+  String get delete {
+    return Intl.message(
+      'Delete',
+      name: 'Delete',
```
Comment (imGok) [inline lib/presentation/localizations/app_localizations.dart:453]: Should be `delete`

File: lib/presentation/localizations/app_localizations.dart [hunk 3]
```diff
@@ -446,6 +446,57 @@ class AppLocalizations {
...
+      args: [documentCount],
+    );
+  }
+
+  String get the_file_is_successfully_deleted {
+    return Intl.message(
+      'The file is successfully deleted',
+      name: 'the_file_is_successfully_deleted',
+    );
+  }
```
Comment (imGok) [inline lib/presentation/localizations/app_localizations.dart:478]: we should have a string for all files have been deleted no  ?

Comment (manhduc-nguyen) [reply lib/presentation/localizations/app_localizations.dart:478]: yes, I've already defined it `some_items_are_successfully_deleted`

File: lib/presentation/localizations/app_localizations.dart (hunk 1 above)
Comment (manhduc-nguyen) [reply lib/presentation/localizations/app_localizations.dart:446]: oh thanks, you save my hands haha

**Review by hoangdat — APPROVED** [review]

**Pull request merged**